python/doc/examples/reliability_sensitivity/estimate_probability_importance_sampling.ipynb

```diff
@@ -4,21 +4,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Probability estimation: importance simulation"
+    "# Probability estimation with importance sampling simulation on cantilever beam example"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this basic example we are going to estimate a failure probability.\n",
+    "In this example we estimate a failure probability with the importance sampling simulation algorithm provided by the `ImportanceSamplingExperiment` class. \n",
+    "\n",
+    "The main steps of this method are:\n",
+    "\n",
+    "* run a FORM analysis,\n",
+    "* create an importance distribution based on the results of the FORM results,\n",
+    "* run a sampling-based probability estimate algorithm. \n",
+    "\n",
+    "\n",
+    "## Description of the problem\n",
     "\n",
     "Let's consider the following analytical example of a cantilever beam, of Young's modulus E, length L, section modulus I.\n",
     "\n",
-    "One end is built in a wall and we apply a concentrated bending load F at the other end of the beam, resulting in a deviation:\n",
-    "$$d = \\frac{F*L^3}{3*E*I}$$\n",
+    "We consider the cantilever beam example; one end is built in a wall and we apply a concentrated bending load F at the other end of the beam, resulting in a deviation:\n",
+    "\n",
+    "$$\n",
+    "d = \\frac{FL^3}{3EI}\n",
+    "$$\n",
     "\n",
-    "It is considered that failure occurs when the beam deviation is too important:\n",
+    "It is considered that failure occurs when the beam deviation is too large:\n",
     "\n",
     "$$d \\ge 30 (cm) $$\n",
     "\n",
```

Comment:
In the next few lines, I think "the" should be systematically deleted.

Reply:
Fixed.